projects/ccf-eui/src/app/app.component.ts

```diff
@@ -25,12 +25,22 @@ export class AppComponent {
    */
   organList = ALL_ORGANS;
 
+  /**
+   * Organs to be selected in the organ selector carousel
+   */
+  selectedOrgans = this.organList.filter(organ => organ.name !== 'Large Intestine');
+
   /**
    * Used to keep track of the ontology label to be passed down to the
    * results-browser component.
    */
   ontologySelectionLabel = 'body';
 
+  /**
+   * Whether or not organ carousel is open
+   */
+  open = true;
```

Comment:
let's refactor this to say `organListVisible`. It'll eventually fall into the state, but will make it more clear.